cryptos-cool/packages/external-microservices/coins/utils.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+import json
+from collections import defaultdict
+import requests
+
+def build_url(type, endtime):
+    # 1 day = 86,400 seconds
+    current_time = datetime.datetime.now().timestamp()
+
+    if endtime <= 7:
+        endtime = current_time - (endtime * 86400)
```

Comment:
Não seria mais interessante utilizar bibliotecas já existentes que te calculam isto?
https://stackoverflow.com/questions/441147/how-to-subtract-a-day-from-a-date

Reply:
pelol timestamp é mais preciso para as outras contas